backend/src/main/java/capstone/herbs/plants/HerbsRepository.java

```diff
@@ -0,0 +1,13 @@
+package capstone.herbs.plants;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface HerbsRepository extends MongoRepository<HerbsItem, String> {
+    Optional<HerbsItem> findAllByHerbsName(String herbsName);
```

Comment:
Wenn du nur ein Item suchst, sollte hier entweder findByID oder findByName genutzt werden. FindAll gibt ne List zurück, das kann nicht in ein Optional gepackt werden.